inst/qml/ConfirmatoryFactorAnalysis.qml

```diff
@@ -80,17 +80,17 @@ Form
 		Group
 		{
 			title: qsTr("Model Options")
-			CheckBox { label: qsTr("Include mean structure")      ; name: "includemeanstructure"   ; id: meanstructure }
-			CheckBox { label: qsTr("Assume factors uncorrelated") ; name: "uncorrelatedFactors"    }
-			CheckBox { label: qsTr("Fix exogenous covariates")    ; name: "fixExogenousCovariates" ; checked: true ; visible: false }
+			CheckBox { label: qsTr("Include mean structure")      ; name: "meanStructure"   ; id: meanstructure }
+			CheckBox { label: qsTr("Assume factors uncorrelated") ; name: "factorsUncorrelated"    }
+			CheckBox { label: qsTr("Fix exogenous covariates")    ; name: "exogenousCovariatesFixed" ; checked: true ; visible: false }
 			DropDown
 			{
 				label: qsTr("Factor Scaling")
-				name: "identify"
+				name: "modelIdentification"
```

Comment:
would it make sense to make the name and label consistent? I think either changing the label to "Model identification" or the name to `factorScaling` would work.